### PR TITLE
Update notes for FastBoot team

### PR DESCRIPTION
**Pull request by danmcclain**
Updates formatting and adds missing notes from the past 6+ months 😳

### CODE REVIEW

**Comment by danmcclain** [comment]
I'm updating the tool I used to export this, do not merge yet please

File: fastboot-team/2016-12/december-16.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# FastBoot Meeting Notes 2016-12-16
+## Attendees
+*Please add yourself.
```
Comment (kratiahuja) [inline fastboot-team/2016-12/december-16.md]: Extra *

File: fastboot-team/2017-01/january-06.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# FastBoot Meeting Notes 2017-01-06
+## Attendees
+*Please add yourself.
```
Comment (kratiahuja) [inline fastboot-team/2017-01/january-06.md]: Ditto

**Comment by danmcclain** [comment]
Fixed, ready to merge

**Comment by rwjblue** [comment]
Thanks Dan!

**Pull request merged**